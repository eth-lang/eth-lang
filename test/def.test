; vim: set ft=clojure:
(def identity (a)
  a)
(def multi ()
  (do-a)
  (do-b)
  10)
