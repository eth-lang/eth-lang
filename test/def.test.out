identity = (function (a) {
    return a;
  });
multi = (function () {
  doA();
  doB();
    return 10;
  });

